### DISCUSSION TITLE
Database triggers?

### DISCUSSION THREAD

**Discussion by rhlsthrm** [prisma/prisma#2382] [Q&A] [answered]
I want to set up database triggers to modify data on INSERTs, UPDATEs, etc to rows. Is there an easy way to do this?

**Comment by Rechdan** [comment]
> I want to set up database triggers to modify data on INSERTs, UPDATEs, etc to rows. Is there an easy way to do this?

I have the same question, if anyone has a answer to this it will be much appreciated!

**Reply by ryands17** [reply, marked as answer]
Hey @rhlsthrm and @Rechdan :wave: 
Currently Prisma doesn't support DB triggers, but there's a proposal for that in the issue referenced [here](https://github.com/prisma/prisma/issues/905).

**Reply by rhlsthrm** [reply]
Thanks! I'm also coming from Hasura, good to know others are having the same issue.

**Comment by ludralph** [comment]
Hi @rhlsthrm,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
